distribution/docker/build.gradle

```diff
@@ -333,11 +332,13 @@ private static List<String> generateTags(DockerBase base) {
     namespace += '-ci'
   }
 
-  return [
-    "${image}:test",
-    "${image}:${version}",
-    "docker.elastic.co/${namespace}/${image}:${version}"
-  ]
+  def tags = ["${image}:${architecture.classifier}"]
```

Comment:
I feel like this should include a version as well? Or is that not necessary?

Reply:
Nope, same reason why it's fine to just use `elasticsearch:test` as these are only used within the scope of a single build invocation.